UnifiedApi/Client/Services/Submitter/Service.cs

```diff
@@ -79,6 +76,37 @@ public class ServiceResult
 
     private Dictionary<string, Task> TaskWarehouse { get; set; } = new();
 
+
+    // *** you need some mechanism to map types to fields
```

Comment:
Yes ! This is called "using the existing types"

Reply:
Idea here is to merge status of task and status of result for HandleError.

Reply:
That will not be the case in protos